NDB.Covid19/NDB.Covid19/Locales/en.json

```diff
@@ -73,13 +73,19 @@
-  "SETTINGS_PAGE_2_CONTENT_TEXT_INTRO": "The Smittestopp contact tracing app is a digital initiative in the fight against the spread of COVID-19.<br><br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_1_TITLE": "Active and inactive status<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_1_CONTENT": "When the app is active, you contribute to stopping the spread of COVID-19. You can always turn registration on or off in the app by pressing the 'Activate' or 'Turn off' button on the home screen. When your app is inactive — as shown by the red background — your phone will stop making registrations.<br><br>When you have given your consent and allowed Bluetooth, the app will operate continuously. You can withdraw your consent under 'Processing of personal data'.<br><br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_TITLE": "Sharing your test result<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_2_CONTENT": "If you have tested positive, you can voluntarily share your status in the app by going to the home screen and pressing ‘Did you test positive?’. Other app users will not be able to see who you are. You must log in via ID-porten.<br><br>After you have shared your status, other users will receive a message saying that they have been nearby a person infected with COVID-19.<br><br>You can see messages in the app on the home screen. Here you can also see if you have received a new exposure message.<br><br>If Smittestopp is inactive, you cannot receive messages. You can log in and share your status, but as there is no exchange of IDs when the app is inactive, there will be nobody to share your status with.<br><br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "The technology<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "The app uses Bluetooth and is based on built-in technology developed by Apple and Google. It comprises a system for registering exposure. You can turn this on or off in your phone’s settings.<br><br>Your phone generates a random Bluetooth ID every 15 minutes. When two phones are nearby each other, they exchange IDs.<br><br> Read more about the Smittestopp app at <a href=\"https://www.helsenorge.no/en/smittestopp\">www.helsenorge.no/en/smittestopp</a><br><br>",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_INTRO": "The Smittestopp contact tracing app is a digital initiative in the fight against the spread of COVID-19.",
```

Comment:
Have you corrected this in the excel sheet?

Reply:
yes